# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/GtkCompositorWidget.h
# Commit: 31f0a398817f
# Full Hash: 31f0a398817fae921abb22b709b5f8328e148aba
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-10 16:36:00
# Regressor Bug: 1934497
# File Overlap Count: 1
# Description:
#   Bug 1934497 [Wayland] Update GtkCompositorWidget for WaylandSurface r=emilio
#   
#   - Replace MozContainerSurfaceLock with WaylandSurfaceLock
#   - Temporary disable GtkCompositorWidget::GetNativeLayerRoot()
#   
# ==============================================================================

diff -r 6225ae4714a9 -r 31f0a398817f widget/gtk/GtkCompositorWidget.h
--- a/widget/gtk/GtkCompositorWidget.h	Tue Dec 10 07:12:54 2024 +0000
+++ b/widget/gtk/GtkCompositorWidget.h	Tue Dec 10 07:12:54 2024 +0000
@@ -11,7 +11,7 @@
 #include "mozilla/widget/CompositorWidget.h"
 #include "WindowSurfaceProvider.h"
 #include "mozilla/UniquePtr.h"
-#include "MozContainerSurfaceLock.h"
+#include "WaylandSurfaceLock.h"
 
 class nsIWidget;
 class nsWindow;
@@ -96,7 +96,7 @@
   void NotifyClientSizeChanged(const LayoutDeviceIntSize& aClientSize) override;
   GtkCompositorWidget* AsGtkCompositorWidget() override { return this; }
 
-  UniquePtr<MozContainerSurfaceLock> LockSurface();
+  UniquePtr<WaylandSurfaceLock> LockSurface();
 
  private:
 #if defined(MOZ_WAYLAND)
